examples/closing-channels/closing-channels.go

```diff
@@ -47,4 +47,10 @@ func main() {
 	// [synchronization](channel-synchronization) approach
 	// we saw earlier.
 	<-done
+
+	// It is possible to read from an empty closed channel,
+	// but instead of waiting for a message, we will
+	// immediately receive a zero value of the channel's type.
+	j := <-jobs
```

Comment:
I think it'd be useful to demonstrate the 2-return form of `<-` here, because it's aimed exactly at this use case